healthmap/mriids2
# Show spinner/loader when user clicks on home page and then QUICKLY navigates to Covid 19 outbreak dropdown

Fixes # http://jira/browse/MRIID-78

# Test case 1:
- Pull down the branch,
- Run the server locally.
- ON THE VERY FIRST LOAD, Make sure spinner/loader shows up when user clicks on home page and then **QUICKLY** navigates to Covid 19 outbreak dropdown. (If you want to see the spinner in action again, reload the home page again)
- Also notice that ON THE VERY FIRST LOAD, chart component for covid cases/deaths is displayed/loaded once **all** the data is fetched via API
- On subsequent loads, there will be no spinner as data is already in Redux state. Similarly there will be no spinner for Ebola outbreak.

# Test case 2:
- run tests using `npm test` followed by typing character `a`.
